tests/integration/__init__.py

```diff
@@ -429,10 +441,10 @@ def is_current_cluster(cluster_name, node_counts, workloads):
     if CCM_CLUSTER and CCM_CLUSTER.name == cluster_name:
         if [len(list(nodes)) for dc, nodes in
                 groupby(CCM_CLUSTER.nodelist(), lambda n: n.data_center)] == node_counts:
-            for node in CCM_CLUSTER.nodelist():
-                if set(node.workloads) != set(workloads):
-                    print("node workloads don't match creating new cluster")
-                    return False
+            #for node in CCM_CLUSTER.nodelist():
```

Comment:
Why this had to be commented out?

Reply:
it's a DSE only related feature, the `workloads` property for a node.
out CCM version doesn't have it. I can add it to our CCM as an empty list, and it might be enough.

also I didn't mentioned it but I've run only the `tests/integration/standard` folder.
all the `tests/unittest` folder is passing.

Reply:
I think this deserves at least a separate patch with explanations ^^^

Reply:
I think anyhow I'll take all the general test related work into it's own PR/patchset
i.e. not of it is related directly to shard awareness